package.json

```diff
@@ -39,9 +39,9 @@
     "open-demo": "opener http://127.0.0.1:3000",
     "open-dev": "npm run dev & npm run open-demo",
     "open-hot": "npm run hot & npm run open-demo",
-    "lint-node": "eslint -c .eslintrc-node *.js demo/webpack.*.js",
-    "lint-react": "eslint --ext .js,.jsx -c .eslintrc-react src demo/*.jsx",
-    "lint-react-test": "eslint --ext .js,.jsx -c .eslintrc-react-test src test/client",
+    "lint-node": "eslint -c .eslintrc-node *.js demo/webpack.*.js --fix",
+    "lint-react": "eslint --ext .js,.jsx -c .eslintrc-react src demo/*.jsx --fix",
+    "lint-react-test": "eslint --ext .js,.jsx -c .eslintrc-react-test src test/client --fix",
```

Comment:
Let's undo the `--fix` and instead have a separate command. My rationale is that in CI, this would "fix" any offending errors in files, pass CI, but the changes **aren't committed to source**.

For dev's you can always do something like:

```sh
$ npm run lint-node -- --fix
```

to add in command line flags for shell commands.

Other side note, I'd be ok with a new command like:

```js
"lint-fix": "npm run lint-node --fix && npm run lint-react --fix && npm run lint-react-test --fix"
```

Reply:
Whoops, left that in accidentally. Pulling out now.